app/src/main/java/org/simple/clinic/sync/SyncConfig.kt

```diff
@@ -8,5 +8,7 @@ data class SyncConfig(
 
     val backOffDelay: Duration,
 
-    val batchSize: Int // Number of records to sync in per network call
-)
+    val batchSizeEnum: BatchSize
```

Comment:
`batchSizeEnum` -> `batchSize`? 

Why did you suffix this with the type?

Reply:
Oh,  there was a lot of code that was still referring to the older `batchSize` property and I didn't want to break that. So I named this `batchSizeEnum` so I could expose the older property without breaking existing code. This will be fixed in the PR that completes sync refactoring.